types/webxr/index.d.ts

```diff
@@ -663,6 +663,19 @@ interface XRPlane {
 
 declare abstract class XRPlane implements XRPlane {}
 
+// Mesh detection
+type XRMeshSet = Set<XRMesh>;
+
+interface XRMesh {
+	meshSpace: XRSpace;
+	vertices: Float32Array;
+	indices: Float32Array;
+	lastChangedTime: number;
+	semanticLabel?: string;
```

Comment:
nit - use spaces instead of tab

```suggestion
    meshSpace: XRSpace;
    vertices: Float32Array;
    indices: Float32Array;
    lastChangedTime: number;
    semanticLabel?: string;
```